src/main/scala/com/chaordicsystems/cactus/Validator.scala

```diff
@@ -0,0 +1,120 @@
+package com.chaordicsystems.cactus
+
+import com.chaordicsystems.cactus.Operator._
+import org.json4s.JsonAST.JNothing
+import org.json4s.{DefaultFormats, JValue}
+
+import scala.util.{Failure, Success, Try}
+
+object Validator {
+  case class OperatorNotProvidedException(m: String = "Operator was not provided. Please check the documentation.") extends Exception(m)
+  case class InvalidOperatorException(op: Any) extends Exception(s"$op is not a valid operator. Please check the project specifications.")
+  case class OperatorHandlerException(op: Operator) extends Exception(s"The operator ${op.toString} wasn't handled as expected.")
+
+  case class FieldNotProvidedException(op: String) extends Exception(s"Operator $op requires a Field and it was not provided. Please check the documentation.")
+  case class InvalidFieldException(field: Any) extends Exception(s"$field is not a valid field. Please check the project specifications.")
+
+  case class ArgsNotProvidedException(op: String) extends Exception(s"Operator $op requires a Args was not provided. Please check the documentation.")
+  case class InvalidArgsException(args: Any) extends Exception(s"$args is not a valid args. Please check the project specifications.")
+
+  case class NotEnoughOperationsException(op: String) extends Exception(s"Operator $op should contain at least two other operations. Please check the documentation for more details.")
```

Comment:
Please remove all these "Please check the documentation" message :-)
It's a bit weird to read this in an exception.